### PR TITLE
STYLE: button component integrated with variants

### PR DESCRIPTION
**Pull request by samyakshah3008**

### CODE REVIEW

File: styles/button/btnhover.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+.outline-error:hover{
+    background-color: var(--danger-color);
+    color: white
+}
+
+.outline-warning:hover{
+    background-color: var(--warning-color);
+    color: white;
```
Comment (MurtazaBagwala07) [inline styles/button/btnhover.css]: use css variables instead

**Review by MurtazaBagwala07 — APPROVED** [review]

File: styles/button/button.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    background-color: white;
+}
+
+.outline-primary{
+    background-color: white;
+    color: var(--primary-color);
+}
+
+.outline-error{
+    background-color: white;
```
Comment (anuj-kumary) [inline styles/button/button.css]: Don't hard code color value use as a variable

**Review by anuj-kumary — CHANGES_REQUESTED** [review]
- Don't hard code color value
- Stick with one CSS units
- Format document using Beautify or Prettier

File: styles/button/btnhover.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    transform: scale(1.1);
+}
+.button-primary:hover{
+    background-color: var(--secondary-color);
+    color: var(--primary-color);
+} 
+.button-secondary:hover{
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+
```
Comment (shilpe26) [inline styles/button/btnhover.css:13]: for better CSS don't give any extra space even at the end line of the css file.

File: styles/button/button.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+.button{
+    background-color: var(--default-color);
+    color: white;
+    min-width: 100px;
+    width: max-content;
+    padding: 0.8rem 1rem;
+    border-radius: 1px;
+    /* border: none; */
+    cursor: pointer; 
+    /* transition-property: all; */
```
Comment (shilpe26) [inline styles/button/button.css]: whenever you feel comfortable remove these unwanted comments

File: styles/button/button.html [hunk 5]
```diff
@@ -60,9 +60,113 @@ <h3>Components</h3>
...
+  style=" margin-top: 50px; width: 761px; height: 294px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+          
+           <h2 class="buttonVariantTitle">Link Buttons</h2> 
+
+
```
Comment (shilpe26) [inline styles/button/button.html:112]: hey! YOu apply so many empty spaces/lines please remove them.

File: styles/documentation.css [hunk 6]
```diff
@@ -48,8 +57,37 @@ a{
...
+
+.buttonVariantTitle{
+    margin-top: 5rem; 
+    color: var(--primary-color);
+}
+
+.btnVariantContainer{
+    height: 100px;
+    /* border: 2px solid black;  */
+    width: 850px;
```
Comment (shilpe26) [inline styles/documentation.css:60]: if you use relative units them is easy for responsive

File: styles/button/button.css (hunk 2 above)
Comment (samyakshah3008) [reply styles/button/button.css]: Sure, done.

File: styles/button/btnhover.css (hunk 3 above)
Comment (samyakshah3008) [reply styles/button/btnhover.css:13]: Sure, thank you! done.

File: styles/button/button.css (hunk 4 above)
Comment (samyakshah3008) [reply styles/button/button.css]: Noted, done.

File: styles/documentation.css (hunk 6 above)
Comment (samyakshah3008) [reply styles/documentation.css:60]: Good point, noted.

**Review by shilpe26 — APPROVED** [review]
Now, it looks good👍🏻you may proceed further.

**Review by anuj-kumary — APPROVED** [review]

**Comment by samyakshah3008** [comment]
Looks good to merge now

**Pull request merged**